### DISCUSSION TITLE
Very few DNS queries to discovery.diyhue.org

### DISCUSSION THREAD

**Discussion by clemone210** [diyhue/diyHue#623] [Q&A] [answered]
Hello,

I have diyHue behind my Pi-hole instance and see very few dns queries to a.m. domain.
Its every minute. For what purpose does it query so often?

Thank you

**Comment by mariusmotea** [comment, marked as answer]
It is used for Hue Essentials bridge discovery, similar to https://discovery.meethue.com/ used by the original bridge.

**Reply by clemone210** [reply]
> It is used for Hue Essentials bridge discovery, similar to https://discovery.meethue.com/ used by the original bridge.

okay thanks a lot!
